scripts/cangen/cangen/can_generator.py

```diff
@@ -220,39 +225,45 @@ def generate_code(bus: Bus, config: Config):
         "bus_name": bus.bus_name,
     }
 
-    logger.debug("Generating code for can messages")
-    _generate_from_jinja2_template(
-        CAN_MESSAGES_TEMPLATE_FILENAME,
-        os.path.join(config.output_dir, bus.bus_name.lower() + CAN_MESSAGES_FILE_NAME),
-        context,
-    )
+    logger.debug("Generating code for can messages and msg registry.")
 
-    logger.debug("Generating code for msg registry")
-    _generate_from_jinja2_template(
-        MSG_REGISTRY_TEMPLATE_FILENAME,
-        os.path.join(config.output_dir, bus.bus_name.lower() + MSG_REGISTRY_FILE_NAME),
-        context,
+    env = _create_jninja_environment(
```

Comment:
I'm going to walk back on what I said earlier about making the env in a function. The function has a fairly long call signature and doesn't help much with the structure. As a reader, I think it's more work to follow this function to it's definition that it would be to just see the environment get made here. Let's get rid of the `_create_jinja_env` and directly run the code here.

For the filters, I like the idea of iterating over a dictionary, but since we have so few elements I think we should just set them directly

```
env.filters["hex"] = hex
env.filters[... = _camel...
```